nuclear.js

```diff
@@ -1,16 +1,20 @@
-import { BehaviorContext } from "./Core/Behavior";
+import {BehaviorContext, BehaviorType} from "./Core/Behavior";
 import BehaviorBuilder from "./Core/BehaviorBuilder";
 import objMgr, { me } from "./Core/ObjectManager";
 import { flagsComponents } from "./Core/Util";
+import Heal from "./Targeting/Heal";
 
 export let availableBehaviors = [];  // Declare global variable for behaviors
-
+export let rootBehavior = null; // Declare global variable for rootBehavior
```

Comment:
`rootBehavior` is owned by Nuclear class and is not a global, nothing should depend on using this.

If there is ever a need to peek into current behaviors Nuclear must expose a public API to do that, making variables globals is the easy and dirty way of solving issues